SOURCES/etc/xapi.d/plugins/netdata.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# this is a XCP-ng plugin. It goes to /etc/xapi.d/plugins/ with the exec bit set.
+# it expects to have the XCP-ng python xapi plugins library installed. (https://github.com/xcp-ng/xcp-ng-xapi-plugins)
+
+import json
+import os
+import shutil
+import sys
+import tempfile
+import urllib2
+
+import XenAPIPlugin
+
+from urlparse import urlparse
+
+sys.path.append('.')
+from xcpngutils import configure_logging, run_command, error_wrapped
+from xcpngutils.filelocker import FileLocker
+
+
+class OperationLocker(FileLocker):
+    def _lock(self):
+        try:
+            # noinspection PyProtectedMember
+            super(OperationLocker, self)._lock()
+        except Exception:
+            raise Exception('The plugin is busy.')
+
+
+netdata_config_content = '''
```

Comment:
I don't think we want to alter the default config through the plugin, since the RPM already provides one.